src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -22,32 +23,50 @@ public void testD_C_O_R_CountIn() {
 	public void testD_C_O_R_CallCheck() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
 		boolean result = rhymer.callCheck();
-		Assert.assertEquals(true, result);
+		Assert.assertTrue(result);
 
 		rhymer.countIn(888);
 
 		result = rhymer.callCheck();
-		Assert.assertEquals(false, result);
+		Assert.assertFalse(result);
 	}
 
+	/// Test nie ma już sensu, ponieważ nowa implementacja stosu ma teoretycznie nieograniczony rozmiar maksymalny
+	/*
 	@Test
 	public void testD_C_O_R_IsFull() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
 		final int STACK_CAPACITY = 12;
 		for (int i = 0; i < STACK_CAPACITY; i++) {
 			boolean result = rhymer.isFull();
 			Assert.assertEquals(false, result);
-			rhymer.countIn(888);
+			rhymer.push(888);
 		}
 
 		boolean result = rhymer.isFull();
 		Assert.assertEquals(true, result);
 	}
+	*/
+
+
+	@Test
+	public void testD_C_O_R_IsFull() {
+		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+		final int STACK_CAPACITY = 12;
+		for (int i = 0; i < STACK_CAPACITY; i++) {
+			boolean result = rhymer.isFull();
+			Assert.assertFalse(result);
+			rhymer.countIn(888);
+		}
+
+		boolean result = rhymer.isFull();
+		Assert.assertFalse(result);
+	}
 
 	@Test
 	public void testD_C_O_R_Peekaboo() {
 		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).